package.json

```diff
@@ -112,7 +112,7 @@
     "fastify-plugin": "^4.5.1",
     "flat": "^5.0.2",
     "helmet": "^7.0.0",
-    "holocron": "^1.8.2",
+    "holocron": "^1.9.1",
```

Comment:
I this required? If I remember correctly the 1.9.0 release was done unintentionally?

Reply:
yes, it includes the fixes for integrity loading in holocron